### PR TITLE
Feature/get transactions cli

### PR DESCRIPTION
**Pull request by motxx**
## What is this pull request?
Interactive-cli for `GetTransactions` API.
   
## Why do you implement it? Why do we need this pull request?
- Remaining task of https://soramitsu.atlassian.net/browse/IR-622
- The implementation of Iroha-cli has been forgotten to be added.

### CODE REVIEW

**Comment by l4l** [comment]
May you please rebase the branch before the review?

**Comment by neewy** [comment]
We need to extend documentation with new API, have we already done it?

File: iroha-cli/interactive/impl/interactive_query_cli.cpp [hunk 1]
```diff
@@ -167,6 +172,22 @@ namespace iroha_cli {
           local_time_, creator_, counter_, account_id);
     }
 
+    std::shared_ptr<iroha::model::Query>
+    InteractiveQueryCli::parseGetTransactions(QueryParams params) {
+      // TODO 22/11/17 motxx - Parser grammer is obvious for user.
```
Comment (l4l) [inline iroha-cli/interactive/impl/interactive_query_cli.cpp]: What do you mean here? make it obvious?

File: iroha-cli/interactive/impl/interactive_query_cli.cpp [hunk 2]
```diff
@@ -167,6 +172,22 @@ namespace iroha_cli {
...
+      // hash1,hash2,... (without spaces)
+      std::stringstream ss(params[0]);
+      GetTransactions::TxHashCollectionType tx_hashes;
+      for (std::string hexhash; std::getline(ss, hexhash, ',');) {
```
Comment (l4l) [inline iroha-cli/interactive/impl/interactive_query_cli.cpp]: [boost](https://stackoverflow.com/a/236976) is more readable imo

File: iroha-cli/interactive/impl/interactive_query_cli.cpp (hunk 1 above)
Comment (neewy) [reply iroha-cli/interactive/impl/interactive_query_cli.cpp]: please mark TODOs in the way that they contain link to GitHub issue or JIRA task

**Comment by motxx** [comment]
> We need to extend documentation with new API, have we already done it?

I submit PR to iroha-api repo.

File: iroha-cli/interactive/impl/interactive_query_cli.cpp (hunk 1 above)
Comment (motxx) [reply iroha-cli/interactive/impl/interactive_query_cli.cpp]: I removed todo, no problem for now.

**Pull request merged**